spec/features/rails_new_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe 'my rails app home page' do
+  it 'shows the welcome message' do
+    visit 'http://localhost:3000'
+    page.text.must_include 'You’re riding Ruby on Rails!'
+    click_link 'Rails Guides'
+    current_path == "http://guides.rubyonrails.org/"
```

Comment:
The true/false result of this line has no effect in your code. Perhaps you'd like to make an assertion here?